client/src/app/journey/manage/manage-journey-version.guard.ts

```diff
@@ -1,5 +1,5 @@
 import { Injectable } from '@angular/core';
-import { Router, CanActivate, CanActivateChild, ActivatedRouteSnapshot } from '@angular/router';
+import { Router, CanActivate, ActivatedRouteSnapshot } from '@angular/router';
 import { JourneyService } from '../journey.service';
 
 @Injectable()
```

Comment:
or just remove    

canActivateChild(route: ActivatedRouteSnapshot) {
     return this.canActivate(route);		     return this.canActivate(route);
   }		   }